### DISCUSSION TITLE
disk partioning

### DISCUSSION THREAD

**Discussion by robertocannito** [SpiralLinux/SpiralLinux-project#21] [General]
Hi how do you suggest to make disk partition?

mb 300 /boot (formatted with wich file system? ext 4?
all the rest for / in btrfs?

is this ok?

**Comment by geckolinux** [comment]
Hi there, no, 300MB `/boot` is too small. Unless you have a specific reason for manual partitioning I would recommend that you stick with one of the automated options in Calamares (Erase disk, Install alongside, Replace partition).

**Reply by robertocannito** [reply]
I tried yesterday but I remember that there wasn't automatic partition, just manual partition, in every case I will check again later when I will be back home.

I've download yesterday iso gnome DE

Thank you.

**Reply by geckolinux** [reply]
Hi, there's definitely multiple automated options. If it only gives the Manual option it would seem like there was a previous failed or cancelled installation attempt during the same session, which leaves open mount points.

**Reply by robertocannito** [reply]
Thank you, I've installed on my two notebook all is perfect and without any problems at the moment.
Installed gnome DE with testing branch

**Reply by geckolinux** [reply]
Great to hear that it's working well for you! Did the upgrade from Stable to Testing go smoothly?

**Reply by robertocannito** [reply]
Yes without any problems
